File: src/backend/pkg/dtos/stats.go
package dtos

type StatsDto struct {
	TotalProjects   int32 `json:"total_projects"`
	TotalIssues     int32 `json:"total_issues"`
	AssignedIssues  int32 `json:"assigned_issues"`
	CompletedIssues int32 `json:"completed_issues"`
	OverdueIssues   int32 `json:"overdue_issues"`
}

File: src/frontend/src/api/stats.ts
import client from "./client";

export interface StatsDto {
    total_projects: number
    total_issues: number
    assigned_issues: number
    completed_issues: number
    overdue_issues: number
}

export function getWorkspaceStats(workspaceId: string): Promise<StatsDto> {
    return client.get<StatsDto>(`/workspaces/${workspaceId}/stats`)
        .then(response => response.data)
}